lib/sentry/client.ex

```diff
@@ -286,7 +288,7 @@ defmodule Sentry.Client do
       event_id: event.event_id,
       culprit: event.culprit,
       timestamp: event.timestamp,
-      message: event.message,
+      message: String.slice(event.message, 0, @max_message_length),
```

Comment:
should we show ... or [TRUNCATED] at the end? or do we not have guidance?

Reply:
We could? The Ruby library does not though https://github.com/getsentry/raven-ruby/blob/120a416cf0df0a08803992a7c643741862f000ad/lib/raven/event.rb#L88